Tests/Fixtures/MatthiasDependencyInjectionTestExtension.php

```diff
@@ -31,6 +45,22 @@ public function load(array $config, ContainerBuilder $container)
 
         // add an alias to an existing service
         $container->setAlias('manual_alias', 'service_id');
+
+        // add an factory service
+        $container->register('manual_factory_service', new Definition());
```

Comment:
@Nyholm   I'm sorry again! The bug is here ... must be <code> $container->register('manual_factory_service', 'stdClass'); 

I will fix and I will push again later.